assets/css/style.css

```diff
@@ -0,0 +1,127 @@
+body {
+    background-image: url("../images/bananas.png")
+}
+
+.top-page {
+    position: relative;
+    bottom: 24px;
+    right: 16px;
+    width: 110%;
+    height: 60px;
+    background-color: #3be;
```

Comment:
if we keep background-color and color and make this css look something like:
    margin-bottom: 20px;
    padding: 10px;
    background-color: #3be;
    color: white;

this can do a lot for us with some other notes I'll make further down.